Library/Formula/yelp-xsl.rb

```diff
@@ -0,0 +1,25 @@
+class YelpXsl < Formula
+  homepage "https://github.com/GNOME/yelp-xsl"
+  url "http://ftp.gnome.org/pub/gnome/sources/yelp-xsl/3.16/yelp-xsl-3.16.1.tar.xz"
```

Comment:
and this:
https://download.gnome.org/sources/yelp-xsl/3.16/yelp-xsl-3.16.1.tar.xz